Terraform doesn't compile

builtin/providers/aws/resource_aws_iam_group_membership_test.go:62: wrong number of args for format in Errorf call: 1 needed but 2 args

Vet found suspicious constructs. Please check the reported constructs
and fix them if necessary before submitting the code for review.
make[1]: *** [vet] Error 1
make: *** [test] Error 2

Addressed by #2299

--- a/builtin/providers/aws/resource_aws_iam_group_membership_test.go
+++ b/builtin/providers/aws/resource_aws_iam_group_membership_test.go
@@ -59,7 +59,7 @@ func testAccCheckAWSGroupMembershipDestroy(s *terraform.State) error {
 		for _, u := range resp.Users {
 			for _, i := range users {
 				if i == *u.UserName {
-					return fmt.Errorf("Error: User (s) still a member of Group (%s)", i, *resp.Group.GroupName)
+					return fmt.Errorf("Error: User (%s) still a member of Group (%s)", i, *resp.Group.GroupName)
 				}
 			}
 		}

